src/apps/bridge/CMakeLists.txt

```diff
@@ -289,7 +289,7 @@ include(${SRC_DIR}/lib/bm_core/cmake/bm_core.cmake)
 setup_bm_ip_stack(LWIP "${BM_CORE_LWIP_INCLUDES}")
 setup_bm_os(FREERTOS "${BM_CORE_FREERTOS_INCLUDES}")
 add_subdirectory(${SRC_DIR}/lib/bm_core bmcore)
-target_include_directories(bmcore PRIVATE ${CMAKE_CURRENT_SOURCE_DIR}/../bmdk_common)
+target_include_directories(bmcore PRIVATE ${CMAKE_CURRENT_SOURCE_DIR})
```

Comment:
While looking into why `bm_debug` was not being included in the unit test builds I found this. I don't think the bridge should be referencing bmdk common files (nor is this the correct path to access it).

TODO - test bridge with this change.

Reply:
Letting a unit run overnight and will observe for any crashes in the morning.

Reply:
No crashes over night, sampling aggregation and logging all worked!